src/components/SearchView.tsx

```diff
@@ -3,11 +3,9 @@ import * as _ from "underscore";
 import { LibraryItem } from "./LibraryItem";
 import { SearchResultItem } from "./SearchResultItem";
 import { LibraryContainer } from "./LibraryContainer";
-import { searchItemResursive, setItemStateRecursive, ItemData } from "../LibraryUtilities";
+import { searchItemResursive, setItemStateRecursive, findAndExpandItemByPath, ItemData } from "../LibraryUtilities";
```

Comment:
This is getting longer, can you consolidate that into just one `LibraryUtilities`?

```js
import * as LibraryUtilities from "../LibraryUtilities";

// Later on...
let stuff = LibraryUtilities. findAndExpandItemByPath(...);
```